crates/bevy_tasks/src/usages.rs

```diff
@@ -1,24 +1,16 @@
-//! Definitions for a few common task pools that we want. Generally the determining factor for what
-//! kind of work should go in each pool is latency requirements.
-//!
-//! For CPU-intensive work (tasks that generally spin until completion) we have a standard
-//! [`ComputeTaskPool`]  and an [`AsyncComputeTaskPool`]. Work that does not need to be completed to
-//! present the next frame should go to the [`AsyncComputeTaskPool`]
-//!
-//! For IO-intensive work (tasks that spend very little time in a "woken" state) we have an IO
-//! task pool. The tasks here are expected to complete very quickly. Generally they should just
-//! await receiving data from somewhere (i.e. disk) and signal other systems when the data is ready
-//! for consumption. (likely via channels)
-
```

Comment:
I'm surprised this is passing CI. I thought we enabled `#[warn(missing_docs)]` on `bevy_tasks`..

Reply:
Well, I just checked that `crates/bevy_tasks/src/lib.rs` indeed starts with `#![warn(missing_docs)]`, but I am too new to all this to understand its consequences and relation to CI. :-)

Reply:
Ah, looking at the rustdoc book, I think this is just because `missing_docs` relates to public API not being documented, whereas the current main status is that there *are* docs that are just "missing" in our human's sense because they're on an internal crate, so they're not rendered. That just appears to be a different kind of "missing docs" than the one that would be caught.

Reply:
I guess we should also resolve this conversation, at least I don't know any actionable next step.